playground-common/update-playground-setups.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Helper script to update all playground setups when we make changes
+# on how playground works (in other words, when we update setup-playground.sh).
+function absPath {
+    python3 -c "import os.path; print(os.path.abspath('$1'))"
```

Comment:
don't need to change, but just fyi i think `realpath` is on glinux by default and does the same thing

Reply:
yea it doesn't exist on macs.